src/tools/spark.md

```diff
@@ -144,172 +47,109 @@ Check out
 [Spark Best Practices](https://robertovitillo.com/2015/06/30/spark-best-practices/)
 for tips on using Spark to its full capabilities.
 
-### `SparkContext` (`sc`)
+Other useful introductory materials:
+* [Spark Programming Guide](https://spark.apache.org/docs/latest/programming-guide.html)
+* [Spark SQL Programming Guide](https://spark.apache.org/docs/latest/sql-programming-guide.html)
 
-Access to the Spark API is provided through `SparkContext`. In the Jupyter
-notebook, this is the `sc` object. For example, to create a
-distributed RDD of monotonically increasing numbers 1-1000:
 
-```python
-numbers = range(1000)
-# no need to initialize sc in the Jupyter notebook
-numsRdd = sc.parallelize(numbers)
-nums.take(10) #no guaranteed order
-```
+### Reading data from BigQuery into Spark
 
-### Spark RDD
+There are two main ways to read data from BigQuery into Spark.
 
-The Resilient Distributed Dataset (RDD) is Spark's basic data structure.
-The operations that are performed on these structures are distributed to
-the cluster. Only certain actions (such as collect() or take(N)) pull an
-RDD in locally.
+#### Storage API
```

Comment:
r? @akkomar for changes to this `spark.md` file.

Reply:
r+.
I should revisit this when we'll have clear plans for the preferred execution platform.